recipes-thirdparty/cobalt-certscope/cobalt-certscope.bb

```diff
@@ -0,0 +1,29 @@
+SUMMARY = "Recipe to update cobalt certscope"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files/:"
+PACKAGE_ARCH = "${APP_LAYER_ARCH}"
+
+SRC_URI += " file://00-cobalt-certscope.conf"
+
+#cobalt-certscope, no need for configuration/compilation
+do_compile[noexec] = "1"
+do_configure[noexec] = "1"
+do_patch[noexec] = "1"
+
+COBALT_CERTSCOPE ?= "${@d.getVar('PRODUCT_CONF_COBALT_CERTSCOPE') or '' }"
```

Comment:
Is `PRODUCT_CONF_COBALT_CERTSCOPE` need to be enabled for all the RDKM builds ?
If so, where is `PRODUCT_CONF_COBALT_CERTSCOPE` defined ?